doc/Git.md

```diff
@@ -0,0 +1,117 @@
+# Git
```

Comment:
Please make sure that `doc/GIT.md` also gets removed within the same commit so that the rename gets detected.